Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Sharing/MarkerDetectorLocalizationSettings.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    public class MarkerDetectorLocalizationSettings : ISpatialLocalizationSettings
+    {
+        public int MarkerID { get; set; }
+        public float MarkerSize { get; set; }
+
```

Comment:
Could this take a reference to the desired marker detector/marker detector type? Then we may be able to use these settings to declare the marker detector compared to just the size and marker id

Reply:
I think this would require some registry of available marker detectors, similar to the registry of available SpatialLocalizers. My thought here was that there can be multiple SpatialLocalizers, one per type of marker detector, but the SpatialCoordinateService they use can all be the MarkerDetectorCoordinateService with a different IMarkerDetector in its prefab.